react/react1/week1/todo_list/src/components/ListItem.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./ListItem.css";
+
+const ListItem = ({ todo, time, id }) => {
+  return (
+    <div>
```

Comment:
Here, the div breaks the ul > li semantics. You don't need to add extra div here. Remove the div.